### ISSUE TITLE
GitIgnore Netbeans IDE and Direnv development files

### ISSUE THREAD

**Issue by dellgreen** [jenkinsci/java-client-api#293]
Exclude Netbeans and Linux development files from being included in the repository
.envrc
nb-configuration.xml

**Comment by khmarbaise** [comment]
Can you make a PR of it I appreciate to integrate that...

**Comment by khmarbaise** [comment]
Can you squash your commits...and mentioned the issue in your commit message like:
```
Fixed #293 - GitIgnore Netbeans IDE and Direnv development files
Optional description
```
Makes it easier for me...
